upgrade.yml

```diff
@@ -575,5 +636,13 @@
   roles:
     - role: horizon
   environment: "{{ env_vars|default({}) }}"
+# Use haproxy port for post upgrade
+- name: set mysql connection string 3307
+  hosts: all:!vyatta-*
+  tasks:
+    - name: set mysql endpoint port 3307
+      set_fact:
+        endpoints:
+          db: "{{ undercloud_floating_ip }}:3307"
 
 - include: site.yml
```

Comment:
As an Operator, it's challenging to have the site.yml run automatically at the end.
If a run fails, it's hard to determine if the failure happened in the upgrade.yml or in the full run of site.yml (since they are doing very similar things).

When I did the Leysin upgrade I found it more simple to remove this line and just do a full run by hand with `-e upgrade=True`.

Perhaps we could call out via a big debug message that full site.yml is about to be run.
Then the operator could know that they can restart with the site.yml if something were to fail.